Code/Editor/MainWindow.cpp

```diff
@@ -12,14 +12,17 @@
 #include <algorithm>
 
 // Qt
-#include <QMenuBar>
-#include <QDebug>
-#include <QMessageBox>
-#include <QInputDialog>
-#include <QHBoxLayout>
 #ifdef Q_OS_WIN
 #include <QAbstractEventDispatcher>
 #endif
+#include <QDebug>
```

Comment:
While you're re-organizing these, I think we can just remove the `QDebug` include (it doesn't look to be used)

Reply:
Done!